pygsti/forwardsims/forwardsim.py

```diff
@@ -373,7 +373,7 @@ def create_layout(self, circuits, dataset=None, resource_alloc=None,
             if 'epp' in array_types:
                 derivative_dimensions = (self.model.num_params, self.model.num_params)
             elif 'ep' in array_types:
-                derivative_dimensions = (self.model.num_params)
+                derivative_dimensions = (self.model.num_params,)
```

Comment:
I thought we had already fixed this at some point? If not, glad we are getting it in now. :)

Reply:
I also thought we fixed it. Not sure what happened with the original intended fix.